broid-kik/src/core/Parser.ts

```diff
@@ -44,13 +44,13 @@ export class Parser {
     if (!normalized || R.isEmpty(normalized)) { return Promise.resolve(null); }
 
     const activitystreams = this.createActivityStream(normalized);
-    activitystreams.actor = {
+    activitystreams.target = {
```

Comment:
This is currently making the tests fail. I do not know the AS2 spec well enough to understand how this is supposed to work, but before flipping this, the integration would pass the `chatId` instead of the `userId` to kik ([here](https://github.com/broidHQ/integrations/blob/a743325026fec939a57e72d831d54ecdc6f24a31/broid-kik/src/core/Adapter.ts#L186)) and the kik library would throw something like:

```
Error: "2e1697fbf6f5a0b979129610e29baf29a1c195b97f9d187bb03262e906ed2ba9" is not a valid username
```